workflows/differentialabundance.nf

```diff
@@ -217,13 +214,15 @@ workflow DIFFERENTIALABUNDANCE {
         }
         .unique()
 
-    ch_all_matrices = VALIDATOR.out.fom
-        .combine(ch_processed_matrices)                         // Add processed marices to what we have in the FOM
+    ch_all_matrices = VALIDATOR.out.sample_meta
+        .combine(VALIDATOR.out.feature_meta.map{ it[1] })
+        .combine(VALIDATOR.out.assays.map{ it[1] })
+        .combine(ch_processed_matrices)
```

Comment:
```suggestion
    ch_all_matrices = VALIDATOR.out.sample_meta              // meta, samples
        .join(VALIDATOR.out.feature_meta)                                // meta, samples, features
        .join(VALIDATOR.out.assays)                                           // meta, samples, features, raw matrix
        .combine(ch_processed_matrices)                                // meta, samples, features, raw, norm, vst
```

No need for the combines and maps- just use a join (these channels have the same meta up to the differential part)